package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "fendersbot",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "nodemon ./bin/www"
```

Comment:
Could this not just be `nodemon index.js`? Also, there are no clear decisions on how this should be hosted. This could be difficult with heroku, not sure what support they have for heroku but I believe they just monitor the process and restart for you.